examples/all-clusters-app/all-clusters-common/src/binding-handler.cpp

```diff
@@ -35,12 +35,14 @@ using chip::Shell::streamer_get;
 using chip::Shell::streamer_printf;
 #endif // defined(ENABLE_CHIP_SHELL)
 
+using namespace chip::app::Clusters;
+
 static bool sSwitchOnOffState = false;
 #if defined(ENABLE_CHIP_SHELL)
 static void ToggleSwitchOnOff(bool newState)
 {
     sSwitchOnOffState = newState;
-    chip::BindingManager::GetInstance().NotifyBoundClusterChanged(1, chip::app::Clusters::OnOff::Id, nullptr);
+    BindingManager::GetInstance().NotifyBoundClusterChanged(1, chip::app::Clusters::OnOff::Id, nullptr);
```

Comment:
If we are changing around namespaces, BindingManager should probably not live in the toplevel `chip::app::Clusters` namespace.  Just put it in `chip::app::Clusters::Bindings`, and at that point could also just call it `Manager`.